snuba/admin/clickhouse/nodes.py

```diff
@@ -18,9 +18,12 @@
 
 
 def _get_local_table_name(storage_key: StorageKey) -> str:
-    schema = get_storage(storage_key).get_schema()
-    assert isinstance(schema, TableSchema)
-    return schema.get_table_name()
+    try:
+        schema = get_storage(storage_key).get_schema()
+        assert isinstance(schema, TableSchema)
+        return schema.get_table_name()
+    except UndefinedClickhouseCluster:
+        return "badcluster"
```

Comment:
should we also filter out this value from what `get_storage_info` returns?

Reply:
I'd rather we query it and just have it blow up